runelite-client/src/main/resources/npc_health.json

```diff
@@ -1160,5 +1160,11 @@
 	"Alchemical Hydra_426": 1100,
 	"Undead Druid_105": 140,
 	"Temple Spider_75": 70,
-	"Sarachnis_318": 400
+	"Sarachnis_318": 400,
+	"Basilisk Youngling_57" : 60,
+	"Monstrous Basilisk_135" : 170,
+	"Typhor_194" : 280,
+	"The Jormungand_363" : 600,
+	"Basilisk Knight_182" : 200,
```

Comment:
after the update today they now have 300hp
![](https://puu.sh/EnVzH/16c8ada7e3.png)

Reply:
Fixed, Should be good now.